src/test/java/games/strategy/util/CoreEqualityComparators.java

```diff
@@ -0,0 +1,69 @@
+package games.strategy.util;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Map;
+
+import games.strategy.test.EqualityComparator;
+
+/**
+ * A collection of equality comparators for core Java and TripleA types.
+ */
+public final class CoreEqualityComparators {
```

Comment:
This class contains implementations of `EqualityComparator` for general Java and TripleA types.  The implementations are identical to what existed prior to this PR.